server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -283,6 +284,46 @@ public class IndicesService extends AbstractLifecycleComponent
         Property.Final
     );
 
+    private static final List<String> INDEXING_DOC_STATUS_DEFAULT_KEYS = List.of(
```

Comment:
rename to INDEXING_DOC_STATUS_DEFAULT_CODES or INDEXING_DOC_STATUS_DEFAULT_REST_CODES?

Reply:
We're removing this setting and instead bucketing the rest status codes as per their category classes (`2xx`, `4xx`, etc).